ui/packages/consul-ui/app/services/routlet.js

```diff
@@ -70,21 +70,11 @@ export default class RoutletService extends Service {
     return key;
   }
 
-  addOutlet(name, outlet) {
-    outlets.set(name, outlet);
-  }
-
-  removeOutlet(name) {
-    outlets.delete(name);
-  }
-
-  // modelFor gets the model for Outlet specified by `name`, not the Route
-  modelFor(name) {
-    const outlet = outlets.get(name);
-    if (typeof outlet !== 'undefined') {
-      return outlet.model || {};
-    }
-    return {};
+  outletFor(routeName) {
+    const keys = [...outlets.keys()];
+    const pos = keys.indexOf(routeName);
+    const key = pos + 1;
+    return outlets.get(keys[key]);
```

Comment:
Here's the new `outletFor` code, which was extracted from the old `addRoute` code which you can see in the red side of the changeset. This approach is what we should be using (and we are now) for getting a specific outlet when you only have access to a routeName.